src/plugins/intel_gpu/src/kernel_selector/kernels/dynamic_quantize/dynamic_quantize_kernel_opt.cpp

```diff
@@ -19,6 +19,11 @@ static std::pair<size_t, size_t> get_input_bf_size(const dynamic_quantize_params
         input_batch = params.inputs[0].Batch().v * params.inputs[0].Feature().v;
     }
 
+    // In Some model, input_f could be dynamic in input0. It refers to IFM value of weight.
+    if (params.inputs[0].is_dynamic() && input_f == 0) {
```

Comment:
could you add an assertion to confirm that `params.fc_ifm_size != 0`?

Reply:
Added